go.mod

```diff
@@ -5,6 +5,7 @@ go 1.12
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/briandowns/spinner v0.0.0-20190319032542-ac46072a5a91
+	github.com/coreos/etcd v3.3.10+incompatible
```

Comment:
Was this intended? This isn't listed as an indirect dependency, but I can't find it actually being used anywhere.

Reply:
Hmm, probably not. I'm not sure where that came from to be honest, removing.